pages/common/gulp.md

```diff
@@ -0,0 +1,11 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
+
+- Run the default task defined inside gulpfile.js:
+
+`gulp`
+
+- Run specific task:
```

Comment:
Yes, it should say "tasks". I feel like explaining what are tasks is out of scope for a tldr page. It's provided in the first example that they are defined in a `gulpfile.js`, the user should seek further information himself by checking the gulpfile or the gulp documentation.